### PR TITLE
added some documentation based on chatlog

### PR DESCRIPTION
**Pull request by coderofsalvation**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,31 @@
 [![CircleCI](https://img.shields.io/circleci/project/gliderlabs/glidergun/release.svg)](https://circleci.com/gh/gliderlabs/glidergun)
 [![IRC Channel](https://img.shields.io/badge/irc-%23gliderlabs-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#gliderlabs)
 
+glidergun allows writing write-once-run-anywhere cli tools using  [Go](https://golang.org) and [Bash](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO.html).
```
Comment (progrium) [inline README.md:6]: actually you don't write Go with glidergun, just Bash

File: README.md [hunk 2]
```diff
@@ -3,11 +3,31 @@
...
 
-	$ curl https://dl.gliderlabs.com/gh/glidergun/latest/$(uname -sm|tr \  _).tgz \
-		| tar -zxC /usr/local/bin
+    $ curl https://dl.gliderlabs.com/gh/glidergun/latest/$(uname -sm|tr \  _).tgz \
+      | tar -zxC /usr/local/bin
+
+
+# How it works 
+
+The `gun`-command will act as a central point for accessing subcommands written in GO+Bash.
```
Comment (progrium) [inline README.md:16]: Again, maybe not mention Go. Go is just used for the glidergun binary. Try dropping the - in `gun`-command

**Comment by progrium** [comment]
Actually I think in general this whole PR will need work, but I'm glad to get some documentation on glidergun finally.

**Comment by coderofsalvation** [comment]
Agreed, feel free to just close this PR and consider it as a
sketch/note-to-self. I just played with it for 10 mins, so I'm probably not
the best person to write the final doc :)

**Pull request closed without merging**